holoviews/element/graphs.py

```diff
@@ -98,6 +119,10 @@ class Graph(Dataset, Element2D):
     kdims = param.List(default=[Dimension('start'), Dimension('end')],
                        bounds=(2, 2))
 
+    _node_type = Nodes
+
+    _edge_type = EdgePaths
```

Comment:
Wondering if this really needs to be private. It might be useful information for users in general though only advanced users should try setting it...